src/TensorFlow.jl

```diff
@@ -165,14 +165,13 @@ library loaded.
 macro py_proc(expr)
     quote
         eval(Main, quote
-            remotecall_fetch($(TensorFlow.load_python_process())) do
+            remotecall_wait($(TensorFlow.load_python_process())) do
```

Comment:
For my reference  why is the python stuff it it's own process?
It is  not being used for speed (since there is a wait).
And it isn't improving stability either.
I've crashed the python hosting worker several times.
Normally it takes out that process and never recovers.
Sometimes the crash immediately cascades into a segfault taking out the main process as well.
(or a OOM)

I guess if it takes out the worker process, user-code could be written to softly crash.
or in the future code could be added to TensorFlow.jl, to reboot the worker once or twice to see if it were a transient fault.